Cargo.toml

```diff
@@ -65,11 +66,13 @@ tokio = { version = "1.45", default-features = false, features = [
 uuid = { version = "1.17", default-features = false, features = [
   "v4",
 ] }
+async-trait = "0.1"
```

Comment:
better use https://github.com/spastorino/dynosaur. We already do so, e.g. in ods